miniupnpd/.gitignore

```diff
@@ -1,5 +1,4 @@
 *.o
-*.d
```

Comment:
We certainly don't want to include dependency files in the git repository.

Reply:
No, this was intended: the dependency files were moved to the `.depend` subdirectory, which was already present in the gitignore file.

Reply:
OK, you removed `*.d`  from .gitignore because it is not needed anymore !